src/apps/user/services.py

```diff
@@ -23,11 +28,11 @@ def register_user(session: Session, user: UserRegisterSchema) -> UserOutputSchem
 
     username_check = session.execute(select(User).filter(User.username == user_data["username"]))
     if username_check.first():
-        raise UserAlreadyExists
+        raise user_already_exists_exception
 
     email_check = session.execute(select(User).filter(User.email == user_data["email"]))
```

Comment:
two queries into one

Reply:
check = select(select(User).where(or_(User.email == user_data["email"], User.username == user_data["username"])))

Reply:
> check = select(select(User).where(or_(User.email == user_data["email"], User.username == user_data["username"])))

@jakub-figat it will not work with the custom service exceptions because I won't be able to differentiate which field data should be placed while raising exceptions